pipeline/price_modelling/modeler_lstm_k_step.py

```diff
@@ -0,0 +1,478 @@
+import os
+import pandas as pd
+import numpy as np
+import tensorflow as tf
+from sqlalchemy import create_engine
+from datetime import datetime
+from datetime import timedelta
+from sklearn import preprocessing
+import matplotlib.pyplot as plt
+from statsmodels.nonparametric.smoothers_lowess import lowess
+import controller as ctr
+
+topicModelTable = 'TopicModel'
+model_start_date = datetime(2010, 1, 1).date()
+
+# Model parameters
+feature_size = 100
+timestep_size = 90
+batch_size = 128
+num_layer = 1
+cell_size = 128
+learning_rate = 0.0005
+epochs = 300
+keep_prob = 0.8
+clipping_cap = 1.0
+training_pct = 0.85
+
+
+def get_sentiment_scored_article():
+    data_dir = os.environ['DATA_DIR']
+    engine = create_engine(
+        'sqlite:///{0}/the_reading_machine.db'.format(data_dir))
+    sentiment_scored_article = pd.read_sql(
+        'SELECT * FROM SentimentScoredArticle', engine,
+        parse_dates=['date'])
+    return sentiment_scored_article
+
+
+def get_topic_modelled_article():
+    data_dir = os.environ['DATA_DIR']
+    engine = create_engine(
+        'sqlite:///{0}/the_reading_machine.db'.format(data_dir))
+    topic_modelled_article = pd.read_sql(
+        'SELECT * FROM {}'.format(topicModelTable), engine)
+    sclae_input = topic_modelled_article.drop('id', axis=1)
+    scaled_topic = pd.DataFrame(preprocessing.scale(sclae_input),
```

Comment:
That's a question more than a comment. Wouldn't be better to use `preprocessing.StandardScaler()`, to apply the same transformation of the training set to the test set?

Reply:
Not sure what you meant here, but the transformation is applied to the input set before the data is split into training and test set. So the same transformation is applied, also, they need to be scaled together, since we are working with time-series where the scale should be the same across time.